packages/diagram/example/simple/.metadata

```diff
@@ -0,0 +1,8 @@
+# This file tracks properties of this Flutter project.
+# Used by Flutter tool to assess capabilities and perform upgrades etc.
+#
+# This file should be version controlled and should not be manually edited.
+
+version:
+  revision: fb86332cd1580ee0a4e02b1fdcc629371caad499
+  channel: widget_image
```

Comment:
heh

Reply:
Heh. Whoops.  I guess it got it from my currently open flutter repo.